### ISSUE TITLE
Release

### ISSUE THREAD

**Issue by rmeissn** [maazl/easytag#1]
It seems this fork of easytag is the most up to date version, fixing several issues. There also seem to be some additions, concluding from the commit messages.

Is it possible to build a release version and to publish it here? Or even better, to build a flatpak and to publish it on flathub? The current flathub build stems from these configs: https://github.com/flathub/org.gnome.EasyTAG

**Comment by maazl** [comment]
Well, I am completely unfamiliar with Flatpak. But if there are contributions that enable a Flatpack build, I can include them. But note that I am not able to test the Flatpack build, neither apply the necessary security fixes of the required libraries regularly.
It is also unclear to me what you mean with a "release version". It need to be linked against the library versions of the specific Linux distribution of the users install target.

**Comment by maazl** [comment]
Indeed. Unfortunately I seem to be the only one that tests this variant of EasyTag. So there is no real definition of "stable" version. ;-)
I try not to check in anything harmful in the master branch and do major changes in (local) feature branches. But this does not prevent bugs in the master, especially because I do not use all (old) features. So the master branch is stable to some degree.

**Comment by leahneukirchen** [comment]
Conversely, making a release is a good way to get bug reports immediately after ;)

Then you just make another release later. =)

**Comment by Kurami32** [comment]
+1

And btw, how do I compile the app? I'm not familiar with this type of environment..

**Comment by maazl** [comment]
A Flatpak release is now available.

**Comment by Kurami32** [comment]
Thank you!
